src/commands/step-functions/aws.ts

```diff
@@ -0,0 +1,120 @@
+import {CloudWatchLogs, StepFunctions} from 'aws-sdk'
+
+import {AWSRequest} from './interfaces'
+
+export const createLogGroup = (cloudWatchLogsClient: CloudWatchLogs, logGroupName: string): AWSRequest => {
+  const params = {
+    logGroupName,
+  }
+
+  return {
+    function: cloudWatchLogsClient.createLogGroup(params),
+  }
+}
+
+export const deleteSubscriptionFilter = (
+  cloudWatchLogsClient: CloudWatchLogs,
+  filterName: string,
+  logGroupName: string
+): AWSRequest => {
+  const params = {
+    filterName,
+    logGroupName,
+  }
+
+  return {
+    function: cloudWatchLogsClient.deleteSubscriptionFilter(params),
+  }
+}
+
+export const enableStepFunctionLogs = (
+  stepFunctionsClient: StepFunctions,
+  stepFunction: StepFunctions.DescribeStateMachineOutput,
+  logGroupArn: string
+): AWSRequest => {
+  const params = {
+    stateMachineArn: stepFunction.stateMachineArn,
+    loggingConfiguration: {
+      destinations: [{cloudWatchLogsLogGroup: {logGroupArn}}],
+      level: 'ALL',
+      includeExecutionData: true,
+    },
+  }
+
+  return {
+    function: stepFunctionsClient.updateStateMachine(params),
+    previousParams: {
+      stateMachineArn: stepFunction.stateMachineArn,
+      loggingConfiguration: stepFunction.loggingConfiguration,
+    },
+  }
+}
+
+export const getStepFunction = async (
```

Comment:
I updated the function names to make the AWS SDK function naming for everything except for `enableStepFunctionLogs`. Since this function is making a specific type of `updateStateMachine` change